### PR TITLE
Update german translation (2020-03-13)

### PR DESCRIPTION
**Pull request by Neui**
~~I'd rather wait for #580 before reviewing and merging just in case there are some strings missing.~~

Possible interesting decisions:
- Treeview Items → Teilaufgaben (subtasks) because Treeview Items is too technical and the treeview items are the (sub)tasks.
- ... in Task Browser → (none) because it is already in a section about the task browser, no need to be redundant.
- shortcuts → Variable (variable) because it fits better (also at first I thought of keyboard shortcuts but they are just quick-access variables)
- The variable descriptions (the one in parenthesis) can't be translated. Not too bad (since it is a developer tool), but maybe it could be considered to make it translatable.

Asking @renehsz for a review (like last time)

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -2194,15 +2197,18 @@ msgstr "@untouched Schlagwort hinzufügen"
 #: GTG/plugins/dev_console.gtg-plugin.desktop:4
 #: GTG/plugins/dev_console/console.py:71 GTG/plugins/dev_console/console.py:100
 msgid "Developer Console"
-msgstr ""
+msgstr "Entwickerkonsole"
```
Comment (renehsz) [inline po/de.po:2200]: Typo here, should be "Entwicklerkonsole"

File: po/de.po [hunk 2]
```diff
@@ -2216,15 +2222,31 @@ msgid ""
 "Type \"help (<command>)\" for more information.\n"
 "\n"
 msgstr ""
+"Willkommen im GTGs Entwickerkonsole\n"
```
Comment (renehsz) [inline po/de.po:2225]: Typo again, should be "in GTGs Entwicklerkonsole"

**Review by renehsz — CHANGES_REQUESTED** [review]
I found a couple of typos that need to be fixed. Otherwise it looks good to me. Thanks again for the work you do!

File: po/de.po (hunk 1 above)
Comment (Neui) [reply po/de.po]: How did I missed that? Fixed.

File: po/de.po (hunk 2 above)
Comment (Neui) [reply po/de.po]: Thanks, fixed.

**Review by renehsz — APPROVED** [review]
Now everything looks great!

**Comment by diegogangl** [comment]
Merged, thanks!

**Pull request merged**